File: cyberdevs.kt
data class CyberDevs(
    val name: String = "CyberDevs",
    val location: String = "Islamabad, Pakistan",
    val specialties: List<String> = listOf("Android Development", "Game Creation", "Cybersecurity")
) {
    fun greet() {
        println("🌌 Welcome to the official repository of $name!")
        println("📍 Based in $location")
        println("⚡ Our Specialties:")
        specialties.forEach { println("  - $it") }
    }
}

fun main() {
    val devs = CyberDevs()
    devs.greet()
}

File: cyberdevs.js
const cyberDevs = {
    name: "CyberDevs",
    location: "Islamabad, Pakistan",
    specialties: ["Android Development", "Game Creation", "Cybersecurity"],
    greet() {
        console.log(`🌌 Welcome to the official repository of ${this.name}!`);
        console.log(`📍 Based in ${this.location}`);
        console.log("⚡ Our Specialties:");
        this.specialties.forEach(spec => console.log(`  - ${spec}`));
    }
};

cyberDevs.greet();
